test/test_util/test_backend.py

```diff
@@ -0,0 +1,79 @@
+# -*- encoding: utf-8 -*-
+import builtins
+import unittest
+import unittest.mock
+
+import pytest
+
+from common.utils.backend import Backend
+
+
+class BackendStub(Backend):
+
+    def __init__(self):
+        self.__class__ = Backend
+
+
+def setup_load_model_mocks(openMock, pickleLoadMock, seed, idx, budget):
```

Comment:
Great to see that this is now a pytest-style unit test!

Reply:
Yes, I hope all that is here is put on pytest!